Resources/views/email/send-invitation/index.html.twig

```diff
@@ -0,0 +1,26 @@
+{% extends '@WebstackUser/email/base.html.twig' %}
+
+{% block greeting %}
+    Bevestig uw account
+{% endblock %}
+
```

Comment:
De map naam hernoemen naar invitation, “send” voegt niks toe aan de map naam. Dat is de actie die wordt uitgevoerd.